src/rpc/net.cpp

```diff
@@ -607,7 +629,7 @@ static const CRPCCommand commands[] =
     { "network",            "ping",                   &ping,                   true,  {} },
     { "network",            "getpeerinfo",            &getpeerinfo,            true,  {} },
     { "network",            "addnode",                &addnode,                true,  {"node","command"} },
-    { "network",            "disconnectnode",         &disconnectnode,         true,  {"node"} },
+    { "network",            "disconnectnode",         &disconnectnode,         true,  {"address", "nodeid"} },
```

Comment:
This is yet another breaking change. Needs rationale and mention in the release notes. Maybe backport.


```
JSONRPCException: Unknown named parameter address (-8)